src/SIS_dyn_cgrid.F90

```diff
@@ -1952,14 +1953,14 @@ subroutine basal_stress_coeff_itd(G, IG, IST, sea_lev, CS)
         do n =1, ncat
           v_i = v_i + vcat(n)**2 / (max(acat(n), CS%puny))
         enddo
-        v_i = v_i - m_i**2
+        v_i = max( (v_i - m_i**2), CS%puny )
```

Comment:
I think that this expression is not dimensionally consistent. CS%puny is described as being non-dimensional (and is used as such on line 1960, but here it is being used in a expression with units of [m2].  Should there be a separate CS%puny_area variable?